### PR TITLE
fix: Add stream feature view in the Web UI

### PR DESCRIPTION
**Pull request by hao-affirm**
**What this PR does / why we need it**:

Show stream feature view in Feast UI
Screen Shot 2022-09-27 at 3 30 28 PM

**Which issue(s) this PR fixes**:

Fixes #

### CODE REVIEW

File: ui/src/utils/custom-tabs/StreamFeatureViewCustomTabLoadingWrapper.tsx [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+
+  const feastObjectQuery = useLoadFeatureView(featureViewName);
+
+  if (
+    feastObjectQuery.isSuccess &&
+    feastObjectQuery.data &&
+    feastObjectQuery.data.type !== FEAST_FV_TYPES.stream
+  ) {
+    throw new Error(
+      `This should not happen. Somehow a custom tab on a ODFV page received data that does not have the shape?`
```
Comment (adchia) [inline ui/src/utils/custom-tabs/StreamFeatureViewCustomTabLoadingWrapper.tsx:34]: Fix this?

File: ui/src/parsers/parseEntityRelationships.ts [hunk 2]
```diff
@@ -88,6 +88,19 @@ const parseEntityRelationships = (objects: FeastRegistryType) => {
    });
  });
 
+  objects.streamFeatureViews?.forEach((fv) => {
+    links.push({
```
Comment (adchia) [inline ui/src/parsers/parseEntityRelationships.ts:92]: you probably also want the batch source linked too right?

Comment (hao-affirm) [reply ui/src/parsers/parseEntityRelationships.ts:92]: yeah this is sth I haven't understand yet. looks like the `streamFeatureView` contains a `batchSource` and a `streamSource`, while the `streamSource` it self also contains a `batchSource`.
So is it guaranteed that each `streamFeatureView` will have this structure of data?

File: ui/src/queries/useLoadRegistry.ts [hunk 3]
```diff
@@ -32,6 +32,7 @@ const useLoadRegistry = (url: string) => {
           return res.json();
         })
         .then<FeatureStoreAllData>((json) => {
+          console.log(json)
```
[the remark this answers is not in the document]
Comment (hao-affirm) [reply ui/src/queries/useLoadRegistry.ts:35]: good catch

**Comment by hao-affirm** [comment]
@adchia updated. let me know if anything needs change. wanna give it a shot for the stream feature view UI

File: ui/src/parsers/parseEntityRelationships.ts (hunk 2 above)
Comment (adchia) [reply ui/src/parsers/parseEntityRelationships.ts:93]: Ah I think you'll actually be fine looks like. Might have been some artifact of before

**Review by adchia — APPROVED** [review]
/lgtm

**Pull request merged**